pkg/cli/aws/client.go

```diff
@@ -43,13 +43,18 @@ type client struct{}
 var _ Client = &client{}
 
 // GetCallerIdentity gets information about the provided credentials.
-func (c *client) GetCallerIdentity(ctx context.Context, region string, accessKeyID string, secretAccessKey string) (*sts.GetCallerIdentityOutput, error) {
-	credentialsProvider := credentials.NewStaticCredentialsProvider(accessKeyID, secretAccessKey, "")
-	stsClient := sts.New(sts.Options{
-		Region:      region,
-		Credentials: credentialsProvider,
-	})
-	result, err := stsClient.GetCallerIdentity(ctx, &sts.GetCallerIdentityInput{})
+func (c *client) GetCallerIdentity(ctx context.Context, region string) (*sts.GetCallerIdentityOutput, error) {
+	// Load the AWS SDK config and credentials
+	cfg, err := config.LoadDefaultConfig(ctx, config.WithSharedConfigProfile("default"))
+	if err != nil {
+		return nil, err
+	}
+
+	// Create an STS client
```

Comment:
What is an STS Client?

Reply:
Security Token Service  Client and it is generally used when short-term access is needed to privileged AWS resources